CMakeLists.txt

```diff
@@ -93,7 +100,12 @@ elseif (${UNIX})
 
     set(PY_SHORT_VERSION "${PY_VERSION_MAJOR}${PY_VERSION_MINOR}")
 
-    find_package(Boost 1.54 REQUIRED COMPONENTS filesystem python-py${PY_SHORT_VERSION})
+    getLibVersion(libboost-python-dev BOOST_PYTHON_VER)
```

Comment:
Why we need so complex solution?

Reply:
Different versions of this lib have its' .so named differently, so for we need this solution in order to find .so correctly

Reply:
I think solution should be easier. This one is really strange. Boost library have support from cmake.

Reply:
This one still utilizes `find_package` from cmake, but the component name for python boost library needs to be specified for `find_package` to work, `getLibVersion` only checks the installed version of the `libboost-python-dev` to determine the correct component name.

Reply:
I shall say that in retrospective this solution was better, here's why:
Name of the Boost.Python bindings varies depending on which package version you install, what kind of maintainer is responsible for it, and thus also depends on the distro.  
![image](https://user-images.githubusercontent.com/43214067/135522573-f50c4f12-15ff-415d-bc57-95cc7cc94c73.png)
I've had a build error on 0.6.0 on Arch and Manjaro: 
```
Could NOT find Boost (missing: python-py39) (found suitable version
  "1.76.0", minimum required is "1.54"
```

Reason: boost bindings are named a bit differently here (in the debug log, it cannot find the Boost.Python .so file, because the dot in "3.9" was missing). It took me 3 days to pinpoint where exactly this error was coming from, and the `getLibVersion` could've saved me a lot of time. I'm leaving this comment for the historical purposes, in case someone else will encounter this build error on 0.6.0 branch.
Fixing it now is probably irrelevant, because Python support will be removed in 0.7.0. (though I plan to commit Arch-based distros initial support, and I have no idea when 0.7.0 will get released, so it may be worth it to get 0.6.0 working on other distros).